lib/BuildSystem/BuildSystem.cpp

```diff
@@ -805,6 +807,15 @@ bool BuildSystemImpl::build(StringRef target) {
   // Create the execution queue.
   executionQueue = delegate.createExecutionQueue();
 
+  // If target name is not passed then we try to load the default target name
+  // from manifest file
+  if (target.size() < 1) {
+    auto defaultTarget = getBuildFile().getDefaultTarget();
+    if (defaultTarget.size() > 0) {
```

Comment:
This can be simplified to just
```
target = getBuildFile().getDefaultTarget();
```
(because if it is empty, target is already empty, so it doesn't change anything).